### ISSUE TITLE
More default options: Paper background and page zoom

### ISSUE THREAD

**Issue by Munzu** [xournalpp/xournalpp#1962]
**Is your feature request related to a problem? Please describe.**
Whenever I open Xournal++, the first thing I have to do every time is 
1. change the background to graph paper.
2. zoom out since it's too much zoomed in for my liking/comfort.

**Describe the solution you'd like**
I'd like more options in `Preferences` > `Defaults`:
- Paper background: Choose from the list of available backgrounds so I can select graph paper as my default
- Page zoom: A percentage slider 

**Describe alternatives you've considered**

- Similarly to `Apply to all pages` in `Journal` > `Paper background`, another option: `Set as default`
- Alternatively, I wish Xournal++ remembered the zoom percentage of my last session.

![defaults](https://user-images.githubusercontent.com/31477160/81572344-6617a780-93a3-11ea-9601-aa3ae39b1db1.jpg)

**Comment by schochet** [comment]
The default paper background can be set in the Journal tab: Journal -> Configure page template -> backgound
The choice set there is remembered and is thereafter used when starting Xournal++ 
However, it is NOT applied automatically to the current document, so you need to exit the program and restart it.
At least that happens for me using version 1.0.18 on linux  (OpenSuse Tumbleweed).

However, I have not found a way to set a default zoom or remember the last zoom setting, which would be very helpful.

**Comment by ngoonee** [comment]
Agreed that default zoom (perhaps 'best fit') would be very useful. There isn't even a keyboard shortcut for the specific zoom options so I always have to either enter the menu or click on the bottom right.

**Comment by vibecoder** [comment]
Related https://github.com/xournalpp/xournalpp/issues/4052

**Comment by vibecoder** [comment]
https://github.com/xournalpp/xournalpp/issues/2137#issuecomment-799956788

**Comment by eekhof** [comment]
I am not sure that there is an option for default page zoom? If I understand correctly, the DPI-scaling setting is there to calibrate the 100% zoom in such a way that the size on the screen is the actual physical size, so changing this would take away the certainty of knowing how large the paper actually is by zooming out to 100% and is thus not suited for use as a "default zoom" option.
